frontend/package.json

```diff
@@ -8,36 +8,40 @@
     "dev-server": "webpack-dev-server"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.10.4",
+    "@popperjs/core": "^2.4.4",
+    "@types/googlemaps": "^3.39.11",
     "autoprefixer": "^9.4.9",
     "axios": "^0.19.0",
     "babel-cli": "6.24.1",
     "babel-core": "6.25.0",
     "babel-loader": "7.1.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-object-rest-spread": "6.23.0",
+    "babel-polyfill": "^6.26.0",
     "babel-preset-env": "1.5.2",
     "babel-preset-react": "6.24.1",
     "bootstrap": "^4.3.1",
-    "css-loader": "^2.1.0",
```

Comment:
I'm having issues building without this, probably need to leave it in?

Reply:
Hmm, I'm not sure -- does it work if you `rm -rf node_moduels && npm install` and try again?

Reply:
Want to say I did that  last night. I retried with the updates pushed, with and without css-loader, and it only seems to compile with css-loader. So I'm thinking this will still be a definite requirement?